website/docs/installation/documentdb/docker.md

```diff
@@ -60,10 +60,80 @@ See [Set up PostgreSQL connection](../../security/authentication.md#set-up-postg
 
 3. See [FerretDB Docker installation](../ferretdb/docker.md) for more details on connecting to FerretDB.
 
-## Development image for PostgreSQL with DocumentDB extension
+### Run development image
 
 The development image for PostgreSQL with DocumentDB extension
 [`ghcr.io/ferretdb/postgres-documentdb-dev`](https://ghcr.io/ferretdb/postgres-documentdb-dev)
 is recommended for debugging problems.
 It includes additional debugging features that make it significantly slower.
 For this reason, it is not recommended for production use.
+
+## Updating to a new version
+
+Before [updating to a new FerretDB release](../ferretdb/docker.md#updating-to-a-new-version), it is critical to install the matching DocumentDB image first.
+
+The following steps are critical to ensuring a successful update.
+
+Edit your Compose file to use the matching DocumentDB image tag.
+You can find the correct tag in the DocumentDB release notes (for example: `17-0.103.0-ferretdb-2.2.0`).
+Then run:
+
+```sh
+docker compose pull <documentdb-container-name>
+docker compose up -d <documentdb-container-name>
+```
+
+Next, from within your `postgres` database, upgrade the DocumentDB extension by running:
+
+```sh
+
+docker compose exec <documentdb-container-name> \
+  psql -U <username> -d postgres -c 'ALTER EXTENSION documentdb UPDATE;'
+```
+
+Replace `<documentdb-container-name>`, `<username>`, `<password>`, and `<host>` as needed.
+
+After the extension update, verify or update `postgresql.conf` settings to ensure the following configurations are present:
+
+```sh
+docker compose exec -T  <documentdb-container-name> \
+  psql -U <username> -d postgres <<EOF
+ALTER SYSTEM SET shared_preload_libraries = 'pg_cron','pg_documentdb_core','pg_documentdb';
```

Comment:
Oh, that's a great find!

Reply:
Yea, when I did this `ALTER SYSTEM SET shared_preload_libraries = 'pg_cron','pg_documentdb_core','pg_documentdb';` it didnt work